show-land/sass/layout/_benefits.scss

```diff
@@ -0,0 +1,60 @@
+@use "sass:list";
+.benefits {
+  padding-top: list.nth($spaces, 16);
+  padding-bottom: 8rem;
```

Comment:
i think you can use the dynamic spacing for padding, margin or gaps.
like this :
`list.nth($spaces, 16)`